core/modules.rs

```diff
@@ -278,6 +282,87 @@ impl<L: Loader> Future for RecursiveLoad<L> {
   }
 }
 
+#[derive(Debug, Clone, PartialEq)]
+#[allow(dead_code)]
+/// Resolved module specifier
+pub struct ModuleSpecifier(Url);
+
+impl ModuleSpecifier {
```

Comment:
I think this was nicer in its own file core/modules_specifer.rs

Reply:
You're right :+1: moved